packages/chrome-extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@clerk/chrome-extension",
+  "version": "0.1.0",
```

Comment:
Do we want to release as a beta version (0.1.0) or go straight to prod? (1.0.0)

Reply:
Let's publish a beta version and use it in our starter to complete our tests and when we are done with it let's publish a production version.